x-pack/plugins/actions/server/builtin_action_types/jira/api.ts

```diff
@@ -128,23 +72,13 @@ const pushToServiceHandler = async ({
     });
   } else {
     res = await externalService.createIncident({
-      incident: {
-        ...incident,
-      },
+      incident,
```

Comment:
The `incident` does not contain the `summary` attribute but the `title` summary. [Jira's service](https://github.com/stephmilovic/kibana/blob/field-mappings-move/x-pack/plugins/actions/server/builtin_action_types/jira/service.ts#L77) check for the summary keyword.

```suggestion
      ...incident, summary: incident.title
```

The above suggestion will fix one the bugs with Jira.